src/GraphQL.Tests/BasicQueryTestBase.cs

```diff
@@ -75,7 +75,7 @@ public ExecutionResult AssertQuery(Action<ExecutionOptions> options, ExecutionRe
                 _.Schema = schema;
                 _.Query = query;
                 _.Root = root;
-                _.Inputs = inputs;
+                _.Variables = inputs;
```

Comment:
Let's rename all `inputs` in args to `variables`.

Reply:
I did a quick case-sensitive search for "inputs" and changed them to "variables" where it seemed appropriate.  Since the `Inputs` class is still called `Inputs`, I did not change it everywhere.

Reply:
OK. `Inputs` class should stay.